index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-array-constructor */
+/* eslint-disable semi */
+const bookTitle = document.getElementById('book-title');
+const authorName = document.getElementById('book-author');
+const button = document.getElementById('submit');
+const list = document.querySelector('#books-ul');
+const array = new Array();
+let counter = 0;
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] [OPTIONAL] Please remove `eslint` comments and try to solve  your problem other way! It will be good for your technical skill.